packages/components/tabs/src/tabs.tsx

```diff
@@ -212,65 +171,59 @@ const Tabs = defineComponent({
       currentName,
       registerPane,
       unregisterPane,
-      nav$,
     })
 
     expose({
       currentName,
-      get tabNavRef() {
-        return omit(nav$.value, ['scheduleRender'])
-      },
     })
 
     return () => {
-      const addSlot = slots['add-icon']
+      const addSlot = slots.addIcon
       const newButton =
         props.editable || props.addable ? (
-          <div
-            class={[
-              ns.e('new-tab'),
-              isVertical.value && ns.e('new-tab-vertical'),
-            ]}
+          <span
+            class={ns.e('new-tab')}
             tabindex="0"
             onClick={handleTabAdd}
-            onKeydown={handleKeydown}
+            onKeydown={(ev: KeyboardEvent) => {
+              if (ev.code === EVENT_CODE.enter) handleTabAdd()
+            }}
           >
             {addSlot ? (
-              renderSlot(slots, 'add-icon')
+              renderSlot(slots, 'addIcon')
             ) : (
               <ElIcon class={ns.is('icon-plus')}>
                 <Plus />
               </ElIcon>
             )}
-          </div>
+          </span>
         ) : null
 
-      const tabNav = () => (
-        <TabNav
-          ref={nav$}
-          currentName={currentName.value}
-          editable={props.editable}
-          type={props.type}
-          panes={panes.value}
-          stretch={props.stretch}
-          onTabClick={handleTabClick}
-          onTabRemove={handleTabRemove}
-        />
-      )
+      const prefixSlot = slots.prefix
+      const prefix = prefixSlot ? (
+        <span class={ns.e('prefix')}>{renderSlot(slots, 'prefix')}</span>
+      ) : null
+
+      const suffixSlot = slots.suffix
+      const suffix = suffixSlot ? (
+        <span class={ns.e('suffix')}>{renderSlot(slots, 'suffix')}</span>
+      ) : null
 
       const header = (
-        <div
-          class={[
-            ns.e('header'),
-            isVertical.value && ns.e('header-vertical'),
-            ns.is(props.tabPosition),
-          ]}
-        >
-          {createVNode(PanesSorter, null, {
-            default: tabNav,
-            $stable: true,
-          })}
+        <div class={[ns.e('header'), ns.is(props.tabPosition)]}>
           {newButton}
+          {prefix}
+          {suffix}
+          <TabNav
```

Comment:
IMO, Arranging them according to the displayed DOM order would be more suitable here.